tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -1065,6 +1065,35 @@ def test_mo_import_from_memory(self, create_model, ie_device, precision, ir_vers
         self._test_by_ref_graph(temp_dir, test_params,
                                 graph_ref, compare_tensor_names=False)
 
+    @ pytest.mark.precommit
+    def test_sharing_memory_switched_off(self, ie_device, precision, ir_version, temp_dir):
+        from openvino.tools.ovc import convert_model
+        from openvino.runtime import Core
+        
+        class DataModel(torch.nn.Module):
+            def __init__(self):
+                super(DataModel, self).__init__()
+                self.data = torch.tensor([1, 2, 3, 4])
+        
+            def forward(self, x):                
+                return x + self.data
+
+        data_model = DataModel()
+        test_input = np.array([1, 2, 3, 4])
+
+        # Convert model to OV
+        ov_model = convert_model(data_model, input=([4], Type.i32), share_weights=False)
```

Comment:
Can we add two cases with share_weights=True and share_weights=False?

Reply:
Added